utils/constants/networking.go

```diff
@@ -27,7 +27,7 @@ const (
 
 	MaxContainersLen = int(4 * DefaultMaxMessageSize / 5)
 
-	DefaultNetworkPeerListNumValidatorIPs        = 15
```

Comment:
Set this so that we will (even in the first message) send some non-bootstrapper IPs so that the first round of connections don't just result in everyone sending the same peerlists (and therefore the nodes not quickly connecting to the wider network)